upgrade_tests/test_existance_relations/test_subscriptions.py

```diff
@@ -24,7 +24,7 @@
 # Required Data
 component = 'subscription'
 sub_name = compare_postupgrade(component, 'name')
-sub_uuid = compare_postupgrade(component, ('id', 'uuid'))
+sub_uuid = compare_postupgrade(component, ('id', 'uuid', 'uuid'))
```

Comment:
why twice uuid

Reply:
if you don't have a unique key in all 6.1, 6.2 and 6.3 satellite, then you have to mention it as a tuple in the format:
``` (6.1_key, 6.2_key, 6.3_key)```
whether its same or different doesn't matter.

These keys are hooks to get component entities data from datastore.